kustomize/base/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: terrascan
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: terrascan-server
+        image: devanggaur7/terrascan:latest
```

Comment:
latest might not be a good choice. even terrascan won't allow that itself. we'll need to fixate on a tag.

accurics/terrascan:1.6.0 does the job.

Reply:
I don’t think that is a good idea since we are making updates every time so it's better to use the latest image to be in sync with new features and if someone wants to stick to a specific tag they can use that but In our example, we should use latest.